server/utils.js

```diff
@@ -209,6 +207,12 @@ function getRandomDate(from, to) {
   return new Date(from + Math.random() * (to - from));
 }
 
+/**
+ * Removes html code tags
+ *
+ * @param {String} str - String to clean
+ * @return {String} str - String without the html tags
+*/
 function convertHtmlToText(str) {
 
   //-- remove BR tags and replace them with line break
```

Comment:
Why do you have ```--``` here? It occurs several times in another comments lines 232,235,238,241....